src/main/java/com/artipie/rpm/hm/NodeHasPkgCount.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2020 artipie.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.artipie.rpm.hm;
+
+import com.jcabi.xml.XMLDocument;
+import org.hamcrest.core.IsEqual;
+import org.llorllale.cactoos.matchers.MatcherEnvelope;
+
+/**
+ * Metadata has given amount of packages.
+ * @since 0.10
+ * @todo #241:30min This matcher should also verify that `packages` attribute of metadata files
+ *  has correct value of packages (the same as packages count). Add this check, correct mismatch
+ *  description to say how many packages was found, what is `packages` attribute value and what is
+ *  expected, do not forget about tests. After that use this matcher in `RpmTest`, `RpmITCase`
+ *  and possibly in ModifiableMetadataTest and MetadataFileTest.
+ */
+public final class NodeHasPkgCount extends MatcherEnvelope<XMLDocument> {
```

Comment:
@olenagerasimova it seems this package should be located in `src/test/java`, not `src/main/java`

Reply:
@g4s8 I do not mind to move this package to test scope, but what we do have matchers in http in main scope, what is the difference here?

Reply:
@olenagerasimova http and asto modules are used by other modules, and they exposes own types, such as `Content`, `Response`, etc, so we need to provide a way for testing these types, e.g. to check the response has correct code. But RPM adapter is self-sufficing module - it's just providing public API using already defined types (`Key`, `Content`, `Slice`), so I don't see the use case for testing XML package nodes outside from RPM adapter.